Sources_v3/StreamChatUI/ChatChannel/ChatMessageImageGallery.swift

```diff
@@ -114,7 +114,7 @@ open class ChatMessageImageGallery<ExtraData: ExtraDataTypes>: View, UIConfigPro
             let attachment = data?.attachments[safe: index]
 
             itemPreview.isHidden = attachment == nil
-            itemPreview.previewURL = attachment?.imagePreviewURL ?? attachment?.imageURL
+            itemPreview.previewURL = attachment?.localURL ?? attachment?.imagePreviewURL ?? attachment?.imageURL
```

Comment:
Perhaps computed property on attachment will be better, in case we need `previewURL` on multiple places